### PR TITLE
Fix bugs and mobile view in feedback page

### LINKED ISSUE #2324: Fix bugs and mobile view in feedback page
**Actual Behaviour**

- Mobile view is not proper
- the unnecessary close button is there
- While loading author name is shown

**Expected Behaviour**

- Fix mobile view
- Remove unnecessary close button
- Author name should be shown after loading 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2324 

Changes: 
- Fix mobile view
- Remove unnecessary close button
- Author name should be shown after loading 

Demo Link: https://pr-2325-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@samagragupta Fixed in #2323 .Close this PR

**Comment by samagragupta** [comment]
@harsh-jindal That PR still does not solve the mobile view problem.
Capture
skip

**Comment by harsh-jindal** [comment]
@samagragupta Mobile view looks good. Resolve the conflicts.

**Comment by samagragupta** [comment]
@harsh-jindal Resolved the conflicts

**Comment by samagragupta** [comment]
@fragm3 Sorry for the mistake.
Made the changes

**Comment by samagragupta** [comment]
@fragm3 Made consistent with the old PR.

**Comment by samagragupta** [comment]
@fragm3 Now its opening.

**Comment by samagragupta** [comment]
@akshatnitd 
Capture

**Comment by akshatnitd** [comment]
Reduce the size of buttons instead in mobile view. Keep them on same line.

**Comment by samagragupta** [comment]
@akshatnitd Done

File: src/components/cms/AuthorSkills/AuthorSkills.js [hunk 1]
```diff
@@ -144,32 +143,35 @@ class AuthorSkills extends Component {
...
+            </Table>
+          </div>
         </div>
       );
     }
 
     return (
       <React.Fragment>
-        <DialogTitle>
-          <CloseButton onClick={actions.closeModal} />
```
Comment (fragm3) [inline src/components/cms/AuthorSkills/AuthorSkills.js]: We need the Close Button

Comment (samagragupta) [reply src/components/cms/AuthorSkills/AuthorSkills.js]: Done

**Comment by samagragupta** [comment]
@fragm3 Made the changes

**Comment by fragm3** [comment]
@akshatnitd do we require the author name after loading?

**Comment by samagragupta** [comment]
@fragm3 Removed that space

**Comment by samagragupta** [comment]
@fragm3 I think the author's name should be loaded.
@akshatnitd @harsh-jindal What are your views?

**Comment by samagragupta** [comment]
@fragm3 And I think the mobile view is good in this PR. If you get any other issue in this PR, then please mention.

**Comment by samagragupta** [comment]
@akshatnitd I have not created the CSS file, it was the previous PR which deleted the CSS file.
Now I have removed the file. Also removed the author's name.

File: src/components/cms/AuthorSkills/AuthorSkills.js [hunk 2]
```diff
@@ -159,13 +147,9 @@ class AuthorSkills extends Component {
 
     return (
       <React.Fragment>
-        <DialogTitle>
+        <AdminTitle>
           <CloseButton onClick={actions.closeModal} />
```
Comment (harsh-jindal) [inline src/components/cms/AuthorSkills/AuthorSkills.js]: No need to keep CloseButton inside Admin Title.

File: src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js [hunk 3]
```diff
@@ -93,6 +93,12 @@ const Pagination = styled.ul`
   align-items: center;
 `;
 
+const PaperContainer = styled(Paper)`
```
Comment (harsh-jindal) [inline src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: Not required. Don't make the padding 0 in mobile view. When you go to write a skill feedback you can see textfield touching the card border.

**Comment by samagragupta** [comment]
@harsh-jindal Made the changes.
Please review.

File: src/components/cms/AuthorSkills/AuthorSkills.js [hunk 4]
```diff
@@ -159,13 +142,7 @@ class AuthorSkills extends Component {
 
     return (
       <React.Fragment>
-        <DialogTitle>
-          <CloseButton onClick={actions.closeModal} />
-          Skills by {author}{' '}
-          <a href={authorUrl && authorUrl !== '<author_url>' ? authorUrl : '/'}>
-            <GithubAvatarImage alt={'GitHub'} src={githubAvatarSrc} />
```
Comment (akshatnitd) [inline src/components/cms/AuthorSkills/AuthorSkills.js]: Why was the author name/url section removed? Add it.

Comment (samagragupta) [reply src/components/cms/AuthorSkills/AuthorSkills.js]: In this he said author name should not be loaded : https://github.com/fossasia/susi.ai/pull/2325#pullrequestreview-253098948

**Comment by fragm3** [comment]
@samagragupta The review above meant Author name need not be loaded after `Circular Loader`

**Comment by samagragupta** [comment]
@akshatnitd @fragm3 Made the changes

**Comment by samagragupta** [comment]
@fragm3 Why it is wrong the author name is loaded along with Circular Loader and after that, it is not loaded.

**Comment by harsh-jindal** [comment]
@samagragupta Still dont see skills by author heading when i click on author name.
Screenshot 2019-06-27 at 8 12 02 AM

**Comment by samagragupta** [comment]
@harsh-jindal I have removed author's name by circular loader.
https://github.com/fossasia/susi.ai/pull/2325#issuecomment-505478254

**Comment by samagragupta** [comment]
@fragm3 Made changes the same as the current version.

**Comment by samagragupta** [comment]
@akshatnitd @harsh-jindal Please review.

File: src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js [hunk 5]
```diff
@@ -83,6 +86,11 @@ const Footer = styled.div`
 
 const PaginationContainer = styled.div`
   padding: 0.875rem;
+  @media only screen and (max-width: 426px) {
```
Comment (fragm3) [inline src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: We do not require this change

File: src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js [hunk 6]
```diff
@@ -105,6 +113,10 @@ const EnabledButtonStyles = css`
   cursor: pointer;
 `;
 
+const AuthorName = styled.span`
```
Comment (fragm3) [inline src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: Use either use class names or styled-components.

File: src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js [hunk 7]
```diff
@@ -51,6 +51,9 @@ const Container = styled(CenterReaderContainer)`
   margin-top: 3.125rem;
   width: 100%;
   font-size: 0.875rem;
+  @media only screen and (max-width: 426px) {
+    padding: 25px 12px;
```
Comment (fragm3) [inline src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: Use rem instead.

File: src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js (hunk 6 above)
Comment (samagragupta) [reply src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: Could not understand what do you mean?

Comment (samagragupta) [reply src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: In this, I have used styled-components

File: src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js [hunk 8]
```diff
@@ -590,12 +597,12 @@ class SkillFeedbackPage extends Component {
                 </SkillName>
                 <div>
                   by{' '}
-                  <span
+                  <AuthorName
                     className="feedback-author"
```
Comment (fragm3) [inline src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: Remove `className`

Comment (samagragupta) [reply src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js]: @fragm3  fixed that

**Comment by samagragupta** [comment]
@fragm3 Done

**Comment by samagragupta** [comment]
@akshatnitd @harsh-jindal Please review
feeeed

**Pull request merged**